Example2/CPP_Abstraction/xt_table_info.cpp

```diff
@@ -666,42 +666,139 @@ compat_ipt_get_target(struct compat_ipt_entry *e)
 }
 
 
-static int compat_calc_entry(const struct ipt_entry *e,
```

Comment:
I think this function needs to remain intact (not changed) and used directly as an API of a C++ XtTableInfo class